docs/api/version-history.md

```diff
@@ -15,7 +15,7 @@ keywords: "API, Docker, rcli, REST, documentation"
 
 ## v1.26 API changes
 
-[Docker Engine API v1.26](v1.26/) documentation
+[Docker Engine API v1.26](v1.26.md) documentation
```

Comment:
Can you change this to https://docs.docker.com/engine/api/v1.26/, and the v1.25 link to  https://docs.docker.com/engine/api/v1.26/ ?

The documentation for docker 1.13 and 1.14 is not published yet, so the links will not be there yet, but once docker they're up, those should be the URL's. See https://github.com/docker/docker.github.io/tree/vnext-engine/engine/api/v1.25